assets/javascripts/discourse/templates/components/babble-composer.hbs

```diff
@@ -1,4 +1,4 @@
 <div class="babble-composer">
-  {{textarea value=text}}
-  <button {{action 'submit'}} {{bind-attr disabled="submitDisabled"}} class="btn btn-primary pull-right">Send</button>
+  {{textarea value=text placeholder=i18nPlaceholder }}
```

Comment:
Oh, I didn't know you can nest those like that - nice! Will do - after my vacation. Thx!